### DISCUSSION TITLE
vsphere-clone - Customize requires Linux Customization Settings

### DISCUSSION THREAD

**Discussion by ghost** [hashicorp/packer-plugin-vsphere#632] [Community Help]
_This issue was originally opened by @FF186 as hashicorp/packer#10041. It was migrated here as a result of the [Packer plugin split](https://github.com/hashicorp/packer/issues/8610#issuecomment-770034737). The original body of the issue is below._

<hr>

Hi, 

I'm wondering why the `customize` options for `vsphere-clone`  requires the `linux_options:` or else it says that `customize is empty`? 
I want the ability to only add `network_interface`.

```
WORKS
 "customize": {
          "linux_options": {
            "host_name": "packer-test",
            "domain": "test.internal"
          },
          "network_interface": {
            "ipv4_address": "10.0.0.10",
            "ipv4_netmask": "24"
          }
 }
```

```
DOES NOT WORK
 "customize": {
          "network_interface": {
            "ipv4_address": "10.0.0.10",
            "ipv4_netmask": "24"
          }
 }
```

**Comment by tenthirtyam** [comment]
For general questions, we recommend reaching out to the [community forum](https://discuss.hashicorp.com/c/packer) for greater visibility.  The GitHub issue tracker is only watched by a small group of maintainers and collaborators and is typically reserved for bug reports and enhancement request. You will typically have more success finding assistance on the forum.

If you find the forum to be more helpful or if you've found the answer to your question elsewhere, please comment and close the issue. If there is no further activity on this question within the next 30 days it will be closed.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by lethargosapatheia** [comment]
This doesn't seem to be a general question though, does it? This looks more like a bug. Apart from the fact that packer doesn't say explicitly what the issue is (I've just come across this looking the error up), even adding "linux_options" and "host_name" isn't enough. Then it will (this time explicitly) complain that the domain is not set, as if that's supposed to be compulsory.
